### ISSUE TITLE
Problem with Minigame

### ISSUE THREAD

**Issue by omochanch** [TheAwesomeGem/fishing-made-better#15]
I am fishing with:
- Iron Fishing Rod
- Long Cast Reel
- Heavy Bobber
- Shiny Hook

When I am in the minigame, I will have the tension bar over the fish and it is actively reeling in the fish. Suddenly, I get kicked out of the minigame without any indicator telling me why. The distance text colour is green but still kicks me out. It will use up my bait but not my durability. This is not a problem with fish less than 90m away. I've been trying to catch a whale but I am not sure why I keep getting kicked out despite other people being able to catch a whale with the same fishing set up.

**Comment by fonnymunkey** [comment]
The long cast reel is slow, and that combined with whales being slow to reel in means its likely taking too long to reel in, however that is supposed to give a popup message on screen when a fish gets away.
 Are you playing on a server? 
 Otherwise, if the bobber entity is being killed somehow, like being knocked onto land or something similar, that could also cause the fishing to stop without a message.

**Comment by omochanch** [comment]
Yes, I'm on a server. We are using the RLCraft package and added [1.12.2] Gravestone Mod 1.10.1. There is no pop up on the screen, I just get kicked out of the minigame. Also I was only reeling for about 10 seconds before it kicks me out. I'm fishing in the middle of the ocean so I don't think its the bobber entity being killed.

**Comment by fonnymunkey** [comment]
Either the server or client being installed incorrectly can cause it to desync the fishing and stopping without a message, that or server performance mods if added, some can cause issues as well.

It only taking 10 seconds before kicking you out of fishing almost sounds like the server either removed the FMB mod, or didnt install any of its configs, I'd recommend reinstalling using the server pack and making sure all the proper configs and mods are there.

**Comment by omochanch** [comment]
Hey! We've redownlaoded all the files and checked the config but are still running into the problem. We also removed the gravestone mod as well. We decided to just turn off the minigame so I can catch the whale as it started kicking me out of the minigame even with other fish :)

**Comment by fonnymunkey** [comment]
Could you upload the latest.log file when this happens, from both the client and server if possible?
